### ISSUE TITLE
loaderUtils.parseQuery() send a DeprecationWarning

### ISSUE THREAD

**Issue by alepee** [Evaneos/vitaminjs#268]
`loaderUtils.parseQuery()` will be replaced by `getOptions()`

```
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
    at Object.parseQuery (/Users/alepee/Documents/Evaneos/community/vitaminjs/examples/counter/node_modules/loader-utils/index.js:78:3)
    at Object.module.exports (/Users/alepee/Documents/Evaneos/community/vitaminjs/examples/counter/node_modules/babel-loader/lib/index.js:86:35)
    at LOADER_EXECUTION (/Users/alepee/Documents/Evaneos/community/vitaminjs/node_modules/loader-runner/lib/LoaderRunner.js:119:14)
    at runSyncOrAsync (/Users/alepee/Documents/Evaneos/community/vitaminjs/node_modules/loader-runner/lib/LoaderRunner.js:120:4)
    at iterateNormalLoaders (/Users/alepee/Documents/Evaneos/community/vitaminjs/node_modules/loader-runner/lib/LoaderRunner.js:229:2)
    at Array.<anonymous> (/Users/alepee/Documents/Evaneos/community/vitaminjs/node_modules/loader-runner/lib/LoaderRunner.js:202:4)
    at Storage.finished (/Users/alepee/Documents/Evaneos/community/vitaminjs/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:38:15)
    at /Users/alepee/Documents/Evaneos/community/vitaminjs/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:69:9
    at /Users/alepee/Documents/Evaneos/community/vitaminjs/node_modules/graceful-fs/graceful-fs.js:78:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:416:3)
```

**Comment by alepee** [comment]
caused by [`babel-loader`](https://github.com/babel/babel-loader/issues/392)

**Comment by alepee** [comment]
according to the issue https://github.com/babel/babel-loader/issues/392 it will be fixed with `babel-loader@7.0.0`

**Comment by christophehurpeau** [comment]
babel-loader@7.0.0 is out !
